probe/kubernetes/persistentvolume.go

```diff
@@ -22,16 +24,34 @@ func NewPersistentVolume(p *apiv1.PersistentVolume) PersistentVolume {
 	return &persistentVolume{PersistentVolume: p, Meta: meta{p.ObjectMeta}}
 }
 
+// GetAccessMode returns access mode associated with PV
+func (p *persistentVolume) GetAccessMode() string {
+	var accessMode string
+
+	if len(p.Spec.AccessModes) > 0 {
+		// A volume can only be mounted using one access mode at a time,
+		//even if it supports many.
+		accessMode = string(p.Spec.AccessModes[0])
+	}
+	return accessMode
+}
+
+// GetVolume returns volume name
+func (p *persistentVolume) GetVolume() string {
+	var volume string
+	if p.Spec.ClaimRef != nil {
+		volume = p.Spec.ClaimRef.Name
+	}
+	return volume
+}
+
 // GetNode returns Persistent Volume as Node
-func (p *persistentVolume) GetNode(probeID string) report.Node {
+func (p *persistentVolume) GetNode() report.Node {
 	return p.MetaNode(report.MakePersistentVolumeNodeID(p.UID())).WithLatests(map[string]string{
-		report.ControlProbeID: probeID,
-		NodeType:              "Persistent Volume",
-		VolumeClaim:           p.Spec.ClaimRef.Name,
-		StorageClassName:      p.Spec.StorageClassName,
-		Status:                string(p.Status.Phase),
-		AccessModes:           string(p.Spec.AccessModes[0]),
-		ReclaimPolicy:         string(p.Spec.PersistentVolumeReclaimPolicy),
-		Message:               p.Status.Message,
+		NodeType:         "Persistent Volume",
+		VolumeClaim:      p.GetVolume(),
```

Comment:
This can be blank; I think it would be better if we just didn't set this field in that case.

Reply:
@bboreham sure! I will fix this in subsequent PR.